src/panels/config/automation/action/types/ha-automation-action-parallel.ts

```diff
@@ -35,9 +37,33 @@ export class HaParallelAction extends LitElement implements ActionElement {
         @value-changed=${this._actionsChanged}
         .hass=${this.hass}
       ></ha-automation-action>
+      <ha-button
```

Comment:
You have the button _after_ the automation action, it should be inside it:

```
<ha-automation-action
    ...
>
   <ha-button ...></ha-button>
</ha-automation-action>
```

Reply:
That did it! Thank you

![image](https://github.com/home-assistant/frontend/assets/94438335/14ec6a38-4fcc-4894-9a66-082d5530c566)